branding/brand.css

```diff
@@ -7,16 +7,8 @@
 
 *********************************************/
 
-:root {
-  --ovirt-masthead-height: 60px;
-  --ovirt-masthead-topborder-size: 4px;
-}
-
 /* --- Application: masthead --- */
 .obrand_masthead {
-  border-top: var(--ovirt-masthead-topborder-size) solid #00659c;
-  height: var(--ovirt-masthead-height);
-
```

Comment:
1. Web Admin will stay on PF3 so it will look differently anyway. 
2. going below the minimum height hardcoded in PF4 masthead will require extra work

Details:
PF4 masthead uses `min-height: var(--pf-c-masthead__content--MinHeight);` and `min-height: var(--pf-c-masthead__main--MinHeight);`. Both vars are defined in `rem` unit (in typical scenario 4.375rem). On my env they resolve to 70 px so the minimum size of the PF4 masthead seems 10px more than in Web Admin ( 60px including top border).

Reply:
Well the framing of webadmin is PF4....kinda ;-)

I'm ok with leaving the height alone, but won't be surprised if it comes up as a question in testing.